### DISCUSSION TITLE
Frontend UI

### DISCUSSION THREAD

**Discussion by swiftugandan** [mediacms-io/mediacms#10] [General]
Is the source code for the front end available? Am only seeing minified js

**Comment by mgogoulos** [comment]
Hello, checkout this: https://github.com/mediacms-io/mediacms/issues/7

**Reply by styiannis** [reply]
> Is the source code for the front end available? Am only seeing minified js

At the moment only the final built files of front end code (js/css) are available as part of MediaCMS code repository. An independent repository is being prepared that will include the entire front end development environment, but it will take some time as we work on some new features that soon will be added to the platform.

**Reply by pxlwrk** [reply]
If you would release the frontend code, the community can help you preparing translation for transiflex.

**Comment by mgogoulos** [comment]
I've updated https://github.com/mediacms-io/mediacms/issues/7 with progress in this space.
